src/Providers/SocialLoginProvider.php

```diff
@@ -26,7 +26,7 @@ function () use ($config) {
                 $providers = [
                     'callback'=> '',
                     'providers'=>[
-                        'Facebook'=> ['enabled'=> true, 'keys'=>['id'=>'1197131063788181','secret'=>'1ee2138ace609549f524945ab9f3db3d']]
```

Comment:
You have to do a git rm or change credentials...
Remember that this project could be public once and there are bots that search for those secret keys and tokens...